common/src/main/java/jp/co/soramitsu/common/view/Extensions.kt

```diff
@@ -5,12 +5,19 @@ import android.widget.TextView
 import jp.co.soramitsu.common.R
 import kotlin.time.ExperimentalTime
 import kotlin.time.milliseconds
+private val TIMER_TAG = R.string.common_time_left
 
 @ExperimentalTime
-fun TextView.startTimer(millis: Long, timeLeftTimestamp: Long = 0L, onFinish: ((view: TextView) -> Unit)? = null): CountDownTimer {
-    val deltaTime = if(timeLeftTimestamp != 0L) System.currentTimeMillis() - timeLeftTimestamp else 0L
+fun TextView.startTimer(millis: Long, timeLeftTimestamp: Long? = null, onFinish: ((view: TextView) -> Unit)? = null) {
```

Comment:
Do we need this onFinish callback? I do not sett where it is used

Reply:
In payout adapter 
```
itemListElementDescriptionLeft.startTimer(timeLeft, timeLeftTimestamp) {
                it.text = context.getText(R.string.staking_payout_expired)
                it.setTextColor(context.getColor(R.color.red))
            }
```